examples/asset-upload/package.json

```
@@ -13,18 +13,18 @@
         "typecheck": "tsc --noEmit"
     },
     "devDependencies": {
-        "@babel/core": "^7.22.11",
-        "@frontify/eslint-config-react": "^0.15.5",
+        "@babel/core": "^7.22.15",
+        "@frontify/eslint-config-react": "0.16.1",
```

```suggestion
        "@frontify/eslint-config-react": "0.16.2",
```

Or is there a problem with that version?

---

Yup, I need to fix an issue with the `prefer-async-to-then` lint rule (Cypress use `.then` but not in a promise context)